app/javascript/readingList/readingList.jsx

```diff
@@ -190,7 +191,7 @@ export class ReadingList extends Component {
         <header className="crayons-layout flex justify-between items-center pb-0">
           <h1 class="crayons-title">
             {isStatusViewValid ? 'Reading list' : 'Archive'}
-            {` (${items.length})`}
+            {` (${itemsTotal})`}
```

Comment:
After spending a while trying to figure out if the backend was broken because I couldn't see the correct total, I found out this counter is broken in production as well: it displays the number of items in the first batch, so up to "page size". What we want to display is the total number of items, regardless of which page we're at